Update alerting-settings.md and add serverless value for one parameter (#250842)
elastic/kibana: docs/reference/configuration-reference/alerting-settings.md

    Data type: `int`
    Default: `10000`
    
    :::{note}
    :applies_to: serverless:

>     In Serverless, the maximum number of rules to run per minute is set to `400`. This setting is not configurable by user. 

Reviewer: ```suggestion
    In Serverless, the maximum number of rules to run per minute is set to `400` and can't be configured.
```
Just to address users directly and not with 3rd person. Thanks for adding this @kunisen
